README.md

```diff
@@ -11,93 +11,124 @@ Master (Release): [![Build Status](https://jenkins.qa.cancercollaboratory.org/bu
 
 ## Documentation
 
-Explore documentation with the Arranger [Read the Docs](https://arranger.readthedocs.io/en/latest/index.html).
+This file is meant as a quick introduction, but for more in-detail documentation, you should explore Arranger's "[Read the Docs](https://arranger.readthedocs.io/en/latest)". If interested, see our Open Source [License](https://github.com/overture-stack/arranger/blob/master/LICENSE)
 
-### Quickstart
+- [Getting Started](#getting-started)
+  - [Development Setup](#--development-setup)
+  - [Dockerized Setup (a.k.a. Quickstart)](#--dockerized-setup)
+- [Motivation](#motivation)
+- [What is a "Data Portal"?](#data-portal)
+- [Roadmap](#roadmap)
+- [Development Details](#development-details)
 
-#### Clone, Install, Bootstrap
+---
 
-```
-git clone git@github.com:overture-stack/arranger.git
-cd arranger
-npm i
-npm run bootstrap
+### Getting Started
 
-# watch modules and rebuild on change
-npm run watch
+#### - Development Setup
 
-# run tests for all modules
-npm test
+Setting up the project, and ready things to make changes
```

Comment:
Minor and optional suggestion "...and prepare things to make changes"

Reply:
Oh I like that!